Packs/FeedBruteForceBlocker/Integrations/FeedBruteForceBlocker/FeedBruteForceBlocker.py

```diff
@@ -0,0 +1,31 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+
+    params['indicator_type'] = FeedIndicatorType.IP
+
+    params['url'] = 'http://danger.rulez.sk/projects/bruteforceblocker/blist.php'
+    params['ignore_regex'] = "^#.*"
+    params['indicator'] = json.dumps({
+        "regex": r"^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}",
+    })
+
+    fields = json.dumps({
+        "upatedate": {
+            "regex": r"(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2})",
+            "transform": "\\1"
+        }
+    })
+    params['fields'] = fields
```

Comment:
What about the `custom_whatever_it's_called` parameter that you added to the httpAPImodule? You haven't included it here.

Reply:
will add after it is merged.

Reply:
lets add that now please, and merge the other PR

Reply:
done